app/boards/shields/m60/m60.keymap

```diff
@@ -0,0 +1,26 @@
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+
+/ {
+	keymap0: keymap {
+		compatible = "zmk,keymap";
+
+		default_layer {
```

Comment:
We've been trying to include at least a layer with the `&bt` bindings *somewhere* for most default keymaps.

Reply:
I tried to keep it simple for this initial PR and add that later. 
But its not much and i think i can add it in this PR itself